modules/aws-outposts-requirements-limitations.adoc

```diff
@@ -8,18 +8,24 @@
 
 You can manage the resources on your Outpost similarly to those on a cloud-based AWS cluster if you configure your {product-title} cluster to accommodate the following requirements and limitations:
 
-* To extend an {product-title} cluster on AWS into an Outpost, you must have installed the cluster into an existing VPC.
+* To extend an {product-title} cluster on AWS into an Outpost, you must have installed the cluster into an existing Amazon Virtual Private Cloud (VPC).
+
+* The infrastructure of an Outpost is tied to an availability zone in an AWS region and uses a dedicated subnet.
+Edge compute machines deployed into an Outpost must use the Outpost subnet and the availability zone that the Outpost is tied to.
+
+* When the AWS Kubernetes cloud controller manager discovers an Outpost subnet, it attempts to create service load balancers in the Outpost subnet.
+AWS Outposts do not support running service load balancers.
+To prevent the cloud controller manager from creating unsupported services in the Outpost subnet, you must include the `kubernetes.io/cluster/unmanaged` tag in the Outpost subnet configuration.
+This requirement is a workaround in {product-title} version {product-version}.
+For more information, see link:https://issues.redhat.com/browse/OCPBUGS-30041[OCPBUGS-30041].
```

Comment:
I've not seen feature docs point to a bug. Can we document the steps in the doc instead of stating the bug? I see this in the event when the bug is resolved and who takes responsibility to check its status to prompt its removal from the doc.

Reply:
We do document the steps in the process, the bug link is just for additional reference